app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -121,7 +121,7 @@ class StateFragmentPresenter @Inject constructor(
 
   private fun subscribeToCurrentState() {
     ephemeralStateLiveData.observe(fragment, Observer<EphemeralState> { result ->
-      logger.d("StateFragment", "getCurrentState: ${result.state.name}")
+      logger.d("StateFragment", "getCurrentState: ${result.state.interaction.customizationArgsMap.get("choices")!!.setOfHtmlString.htmlList}")
```

Comment:
I suspect this was here for debug purposes, but it's a misleading log statement to check in. Please revert.

Reply:
Reverted this change.